ionc/ion_stream.c

```diff
@@ -141,18 +141,21 @@
 #elif defined(ION_PLATFORM_WINDOWS)
   #define LSEEK _lseek
 #else
+  #include <unistd.h>
```

Comment:
These blocks should probably be unified, but basically there's three separate checks to see if we're windows or not and then if we're Linux-ish we define some macro aliases for posix APIs but never actually include the proper headers.